sources/org.osbuild.files

```diff
@@ -71,8 +83,18 @@ def main(options, checksums, cache, output):
 
     with concurrent.futures.ProcessPoolExecutor(max_workers=15) as executor:
         requested_urls = []
+        rhsm_secrets = None
+
         for checksum in checksums:
             try:
+                secrets = urls[checksum].get("secrets")
+                # if the secret is for rhsm replace the secrets value with the rhsm certs
+                if secrets == "org.osbuild.rhsm":
+                    if rhsm_secrets is None:
+                        rhsm_command = ["python3", "secrets/org.osbuild.rhsm"]
```

Comment:
Setting the +x bit for `org.osbuild.rhsm` and just calling it without `python3` should be fine, no?

Reply:
One other thing, I don't think it is safe to assume that the current working directory is so that `secrets/org.osbuild.rhsm` will always work. I think long-term we might wanna pass the `libdir` to the source binary and then it can resolve the `secrets` directory via that. For now, I suggest to use `__file__` to get the path to the current source and then assume that the `secrets` are located next to the `sources` directory in which the current source is; i.e. `os.path.join(os.path.dirname(os.path.dirname(__file__)), "secerts/org.osbuild.rhsm")` or something along those lines.

Reply:
The file path is now set relative to `__file__`

Reply:
org.osbuild.rhsm is set with +x